pages/welcome/welcome.py

```diff
@@ -3,160 +3,59 @@
 import dash_bootstrap_components as dbc
 import plotly.express as px
 
+# import other sections
+from .sections.general_section import layout as general_tab_contents
+from .sections.plotly_section import layout as plotly_tab_contents
+from .sections.augur_login_section import layout as augur_tab_contents
+
 # register the page
 dash.register_page(__name__, path="/", order=1)
 
-# This is just a stand-in until we wire up a more interesting visualization.
-example_fig = px.scatter(
-    x=[1, 2, 3, 4, 5, 6],
-    y=[1, 2, 3, 4, 5, 6],
-    color=["blue", "red", "green", "yellow", "purple", "brown"],
-    labels={"x": "X Values", "y": "Y Values"},
-    size_max=19,
-    size=[1, 1, 1, 1, 1, 1],
-)
-
-example_fig.update_layout(
-    title={
-        "text": "Example Figure",
-        "y": 0.9,
-        "x": 0.5,
-        "xanchor": "center",
-        "yanchor": "top",
-    }
-)
 
 layout = dbc.Container(
     className="welcome_container",
     children=[
         html.Div(
-            className="welcome_message welcome_section",
+            className="toplevel_welcome_div",
             children=[
-                # html.H1("Welcome to 8Knot"),
-                html.Img(src="assets/logo-color.png"),
-                html.P(
-                    """
+                html.Div(
+                    className="welcome_callout_section",
+                    children=[
+                        html.Img(src="assets/logo-color.png"),
+                        html.P(
+                            """
                     Open source communities are difficult to understand. 8Knot serves community stakeholders by
```

Comment:
Not sure how I feel about the first sentence. Could see taking it out or changing it to something like "Open source communities are complex and full of nuance." Also trying to think of a different word for "stakeholders", to me it gives a corporatey vibe

Reply:
++ very fair. This has felt like a placeholder for a while since we haven't done any marketing-copy stuff. I'll make it more concise but we should loop back purposefully.